test/0_staking.js

```diff
@@ -10,22 +11,53 @@ const expect = require('chai')
     .use(require('chai-bn')(BN))
     .use(require('chai-as-promised'))
     .expect;
-const ValidatorSetContract = require('../utils/getContract')('ValidatorSetAuRa', web3);
-const StakingTokenContract = require('../utils/getContract')('StakingToken', web3);
+const getContract = (((getContractInner, w) => name => getContractInner(name, w))(require('../utils/getContract'), web3));
+const ValidatorSetContract = getContract('ValidatorSetAuRa');
+const StakingTokenContract = getContract('StakingToken');
+const BlockRewardContract = getContract('BlockRewardAuRa');
 const canStakeAndWithdraw = require('../utils/canStakeAndWithdraw');
 const pp = require('../utils/prettyPrint');
 
 describe('Candidates make stakes on themselves', () => {
+    'use strict';
     var minStake;
     var minStakeBN;
     before(async () => {
         minStake = await ValidatorSetContract.instance.methods.getCandidateMinStake().call();
         minStakeBN = new BN(minStake.toString());
     });
 
+    it('makes the owner the ERC20 to Native Bridge', async () => {
+        const tx = await SnS(web3, {
+            from: OWNER,
+            to: BlockRewardContract.address,
+            method: BlockRewardContract.instance.methods.setErcToNativeBridgesAllowed([OWNER]),
+            gasPrice: '0x0',
+        });
+        console.log(tx);
+        pp.tx(tx);
```

Comment:
`pp.tx(tx)` is just a less-detailed pretty-print of `console.log(tx)`, so only one of them should be kept. I'd say, leave pp.tx because it's easier to read.
(Also in `Owner issues initial balances to candidates` test)

Reply:
Fixed